samples/AspNetIntegration/AspNetIntegration.csproj

```diff
@@ -9,14 +9,11 @@
     
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.17.4" />
-  </ItemGroup>
-  
-  <ItemGroup>
-    <ProjectReference Include="..\..\extensions\Worker.Extensions.Abstractions\src\Worker.Extensions.Abstractions.csproj" />
-    <ProjectReference Include="..\..\extensions\Worker.Extensions.Http\src\Worker.Extensions.Http.csproj" />
-    <ProjectReference Include="..\..\extensions\Worker.Extensions.Storage\src\Worker.Extensions.Storage.csproj" />
-    <ProjectReference Include="..\..\extensions\Worker.Extensions.Http.AspNetCore\src\Worker.Extensions.Http.AspNetCore.csproj" />
-    <ProjectReference Include="..\..\src\DotNetWorker\DotNetWorker.csproj" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Abstractions" Version="1.3.0" />
```

Comment:
Similar question as for blobs and tables - confirming we want this to be a package reference here, in contrast to most of the other extensions.

Reply:
Commented above.